tests/test_save_identity.py

```diff
@@ -7,11 +7,13 @@
 
 class TestApi(TestCase):
     def test_save_identity(self):
-        payload = {
-            'identity': '0x0000000000000000000000000000000000000001',
+        payload = {}
+
+        headers = {
+            "Authorization": "Signature Da1mAwK5abmXQCNsCE+YjsZbR9jTyEKqdrjxxMKwNzwr2NFnM35UiVQJWcg8rgL+X2PR60LoIUMlGU9OPaSoZwE="
         }
 
-        re = self.client.post('/v1/identities', data=json.dumps(payload))
+        re = self.client.post('/v1/identities', headers=headers, data=json.dumps(payload))
```

Comment:
Can we assert that new identity was created? This can be done using `db.Identity` - getting `Identity` by `identity` string field.